test/infrastructure/docker/api/v1alpha3/condition_consts.go

```diff
@@ -63,3 +63,20 @@ const (
 	// transient and failed bootstrap are automatically re-tried by the controller.
 	BootstrapFailedReason = "BootstrapFailed"
 )
+
+// Conditions and condition Reasons for the DockerCluster object
+
+const (
+	// LoadBalancerProvisionedCondition provide an observation of the provisioning of the
+	// container that implements the cluster load balancer.
+	//
+	// NOTE: When the load balancer provisioning starts the process completes almost immediately and within
+	// the same reconciliation, so the user will always see a transition from no condition to Provisioned without
+	// having evidence that the operation is started/is in progress.
+	LoadBalancerProvisionedCondition clusterv1.ConditionType = "LoadBalancerProvisioned"
```

Comment:
Rather than `LoadBalancerProvisioned`, would it make sense to be`LoadBalancerAvailable` instead?

Reply:
+1 to express the conditions with available states, rather than create/delete/provisioned/etc

Reply:
done!